es-core/src/MameNames.h

```diff
@@ -9,10 +9,11 @@ class MameNames
 {
 public:
 
-	static void       init       ();
-	static void       deinit     ();
-	static MameNames* getInstance();
-	std::string       getRealName(const std::string& _mameName);
+	static void              init       ();
+	static void              deinit     ();
+	static MameNames*        getInstance();
+	const bool               isAsset    (const std::string& _mameName);
+	std::string              getRealName(const std::string& _mameName);
```

Comment:
no need for styling changes here. if you think the indent is wrong, it can be done as a separate later commit.